valispace/__init__.py

```diff
@@ -169,6 +187,43 @@ def update_vali(self, name=None, id=None, formula=None, data={}):
 
 		return result
 
+	def post_data(self, type=None, data={}):
+		# Post new component/vali/textvali/tags with the input data
+
```

Comment:
Add in the comment some description of how the input should be formatted? Json format with at least name or something like that?